Sources/Plasma/PubUtilLib/plGLight/plLightInfo.cpp

```diff
@@ -173,6 +172,13 @@ void plLightInfo::GetStrengthAndScale(const hsBounds3Ext& bnd, float& strength,
     return;     
 }
 
+bool plLightInfo::AffectsBound(const hsBounds3Ext& bnd)
+{
+    if (plVolumeIsect* isect = IGetIsect())
```

Comment:
```suggestion
    if (plVolumeIsect* isect = IGetIsect(); isect)
```

Silences warnings about `=` in an `if` condition